app/controllers/cinema_halls_controller.rb

```diff
@@ -1,4 +1,47 @@
 # frozen_string_literal: true
 
 class CinemaHallsController < ApplicationController
+	def index 
+		@cinema_halls = CinemaHall.all
+		render json: @cinema_halls
+	end
+
+	def create
+		@cinema_hall = CinemaHall.new(cinema_hall_params)
+
+		if @cinema_hall.save
+			render json: @cinema_halls,
+				status: :created, location: @cinema_hall 
```

Comment:
I've never created Rails application with views but I think `location` is redundant because it is connected with views you do not have :) In `render` should be just returned data as `json` and response status code `status`.